README.md

```diff
@@ -142,25 +142,16 @@ py -3 -m venv ./.venv
      ```bash
      python robot.py sim
      ```
-     python -m robotpy_installer install-python
-
-   ```
-
-   ```
-
-1. **Upload robotpy modules to roboRIO**
-   (must be connected to roboRIO)
-   ```bash
-   python -m robotpy_installer install robotpy
-   ```
-   (examples: `robotpy`, `robotpy[ctre,navx]`, `robotpy[all]`) (see: [robotpy on pypi](https://pypi.org/project/robotpy/))
-1. **Deploy robotpy program**
```

Comment:
Why are the deploy/robot install steps removed?

Reply:
Because they were duplicated in the README.md file.  I'm guessing a cut/paste error, if you look they are still there above this section.